src/modules/client/components/planning/ChipAuxiliaryIndicator.vue

```diff
@@ -132,6 +130,7 @@ export default {
       try {
         this.monthHoursDetails = await Pay.getHoursBalanceDetail({ auxiliary: this.person._id, month });
         const monthCustomersDetails = await Stats.getPaidInterventionStats({ auxiliary: this.person._id, month });
+        // eslint-disable-next-line prefer-destructuring
         this.monthCustomersDetails = monthCustomersDetails[0];
```

Comment:
si tu n'aimes pas cette règle tu peux disable pour les array.
Ici c'est le cas typique : 
**[** this.monthCustomersDetails **]** = monthCustomersDetails;

Reply:
tu ss raison - mon cerveau est un peu ramoli ^^